src/cmd/src/frontend.rs

```diff
@@ -263,10 +264,11 @@ impl StartCommand {
 
         let mut instance = FrontendBuilder::new(
             cached_meta_backend.clone(),
+            catalog_manager.clone(),
             Arc::new(DatanodeClients::default()),
             meta_client,
         )
-        .with_cache_invalidator(cached_meta_backend)
+        .with_cache_invalidator(catalog_manager.clone())
```

Comment:
I am going to try refactoring the `cache invalidator` in next PR since it has some problems right now:
1. The catalog manager is a `Cache invalidator`, it has some high level cache
2. The KvBackend is also a `Cache invalidator`, it has the cache of KVs

When we want to invalidate the cache, the metasrv send messages via broadcast to call the KvBackend's `Cache invalidator`. But the Frontend invalidate caches by calling the catalog manger's `Cache invalidator`
That it to say: the metasrv can not invalidate the high level cache.
I had fix it in some way looks not so good now, I will send an another PR ASAP